_layouts/default.html

```diff
@@ -1,159 +1,192 @@
+        </p>
       </div>
     </div>
-    <div class="container main outer">
+  </div>
+  </div>
+  <div class="container main outer">
+    <div class="row">
+      {% comment %}
+      vyom:210623
+      upwork project https://www.upwork.com/jobs/Renovate-static-blog-HTML-and-CSS_%7E01065076083bfce051
+      Removed sidebar
+      {% endcomment %}
+      {% comment %} <div class="col-md-3 col-xs-12">
+        {% include sidemenu.html %}
+      </div> {% endcomment %}
+
+      {% comment %}
+      vyom:210623
+      upwork project https://www.upwork.com/jobs/Renovate-static-blog-HTML-and-CSS_%7E01065076083bfce051
+      Made content full width
+      {% endcomment %}
+      <div class="col-xs-12 full">
+        {{ content }}
+
         <div class="row">
-            <div class="col-md-3 col-xs-12">
-                {% include sidemenu.html %}
-            </div> <!-- end /.col-md-3 -->
-
-            <div class="col-md-9 col-xs-12 full">
-                {{ content }}
-
-                <div class="row">
-                  <div class="col-md-12 col-xs-12 footer">
-                <footer>
-                  © Vladimir Feinberg 2016, 2017, 2018, 2019, 2020, 2021. Favicon by <a href="https://freeicons.io/profile/714">Raj Dev</a>.
-                </footer>
-            </div>
-                </div>
-            </div> <!-- end /.col-md-9 -->
-        </div> <!-- end /.row -->
-
-    </div> <!-- end /.container -->
-
-        <!-- Bootstrap core JavaScript
+          <div class="col-md-12 col-xs-12 footer">
+            <footer>
+              © Vladimir Feinberg 2016, 2017, 2018, 2019, 2020, 2021. Favicon by <a
+                href="https://freeicons.io/profile/714">Raj Dev</a>.
+            </footer>
+          </div>
+        </div>
+      </div> <!-- end /.col-md-9 -->
+    </div> <!-- end /.row -->
+
+  </div> <!-- end /.container -->
+
+  <!-- Bootstrap core JavaScript
         ================================================== -->
-        <!-- Placed at the end of the document so the pages load faster -->
-        
-        <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
-        <script src="//code.jquery.com/jquery-migrate-1.2.1.min.js"></script>
-        <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
-        
-
-        <script src="{{ site.baseurl }}/static/js/thickbox-compressed.js"></script>
-        <script src="{{ site.baseurl }}/static/js/material.min.js"></script>
-        <script src="{{ site.baseurl }}/static/js/main.js"></script>
-        <script src="{{ site.baseurl }}/static/js/projects.js"></script>
-    </body>
-</html>
+  <!-- Placed at the end of the document so the pages load faster -->
+
+  <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+  <script src="//code.jquery.com/jquery-migrate-1.2.1.min.js"></script>
+  <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
```

Comment:
I can see it was this way before, but is url supposed to start with `//*` rather than `https://`?